apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1012,6 +1016,17 @@ fields("log_burst_limit") ->
                 }
             )}
     ];
+fields("log_throttling") ->
+    [
+        {"window_time",
+            sc(
+                emqx_schema:duration_s(),
+                #{
+                    default => <<"1m">>,
+                    desc => ?DESC("log_throttling_window_time")
```

Comment:
missing desc and label in `rel/i18n/emqx_conf_schema.hocon`

Reply:
Fixed, thanks.